src/main/java/com/kamycz3q/coursesystemspringboot/personalData/controllers/PersonalDataCompanyController.java

```diff
@@ -0,0 +1,50 @@
+package com.kamycz3q.coursesystemspringboot.personalData.controllers;
```

Comment:
Subdomena personalData? Dane osobowe są raczej częścią innych bytów. Przy określaniu granic w projekcie trzeba się zastanowić co jest *agregatem*, czyli agreguje (zawiera) inne byty (takie jak PersonalData). W tym przypadku to raczej Customer agreguje PersonalData.

Reply:
Subdomena personalData przeniesiona do customera.